pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computefirewallpolicyrulefull/_http.log

```diff
@@ -723,7 +710,6 @@ X-Xss-Protection: 0
   "name": "network-${uniqueId}",
   "networkFirewallPolicyEnforcementOrder": "AFTER_CLASSIC_FIREWALL",
   "routingConfig": {
-    "bgpBestPathSelectionMode": "LEGACY",
```

Comment:
`routingConfig.bgpBestPathSelectionMode` is a field belongs to ComputeNetwork. Per the [doc](https://cloud.google.com/compute/docs/reference/rest/beta/networks), this field is `The BGP best path selection algorithm to be employed within this network for dynamic routes learned by Cloud Routers. Can be LEGACY (default) or STANDARD.` However, I do not see this field in the [proto](https://raw.githubusercontent.com/googleapis/googleapis/refs/heads/master/google/cloud/compute/v1/compute.proto)...

I feel it's fine to skip this diff, since network is just a reference of the policy rule re source.